pkgs/top-level/python-packages.nix

```diff
@@ -4991,12 +4991,18 @@ self: super: with self; {
 
   jaxlib-bin = callPackage ../development/python-modules/jaxlib/bin.nix {
     cudaSupport = pkgs.config.cudaSupport or false;
+    # 2023-05-11: fails on the error below with cudaPackages_11_8, drop asap
+    # cp: cannot stat '/nix/store/...-cudatoolkit-11.8.0-merged/include/./include': No such file or directory
+    cudaPackages = pkgs.cudaPackages_11_7;
```

Comment:
```suggestion
```

Should have been fixed in https://github.com/NixOS/nixpkgs/pull/234053; 11.8 should work.